wheel/generate_type_stubs.py

```diff
@@ -338,7 +354,7 @@ class BLSCache:
     def __init__(self, cache_size: Optional[int] = 50000) -> None: ...
     def len(self) -> int: ...
     def aggregate_verify(self, pks: List[G1Element], msgs: List[bytes], sig: G2Element) -> bool: ...
-    def items(self) -> List[Tuple[bytes, bytes]]: ...
+    def items(self) -> List[Tuple[bytes, GTElement]]: ...
```

Comment:
I don't think this change would work. We still pass this type across the python process boundary, so it needs to be pickled.

Reply:
GTElement is a Rust pybinding in Python right? So passing it across should be straightforward. Or am I missing something?

Reply:
the way current python code is written, suggests that we have to manually convert objects into something that supports pickle.
looking at `crates/chia_py_streamable_macro/src/lib.rs`, it seems like we implement the pickle protocol in our bindings though. So, as far as I can tell, this ends up serializing the `GTElement` into bytes.
I don't believe we have any test coverage or round-trips of the pickle support though